### PR TITLE
Feature/nguyt141 restructure

### PR DESCRIPTION
**Pull request by nguyt120**

### CODE REVIEW

File: fum_flows/models/integration/fum_flow_whole_bank.sql [hunk 1]
```diff
@@ -0,0 +1,339 @@
...
+    || "|" || CAST(payid_transaction_amount AS STRING)
+    || "|" || CAST(interest_transaction_count AS STRING)
+    || "|" || CAST(interest_transaction_count AS STRING)
+    || "|" || CAST(payto_transaction_count AS STRING)
+    || "|" || CAST(payto_transaction_amount AS STRING)
+    || "|" || CAST(deposit_withdrawal_transaction_count AS STRING)
+    || "|" || CAST(deposit_withdrawal_transaction_amount AS STRING)
+    || "|" || CAST(unidentified_transaction_count AS STRING)
+    || "|" || CAST(unidentified_transaction_amount AS STRING)
+    ) hash_diff,
```
Comment (nguyea57) [inline fum_flows/models/integration/fum_flow_whole_bank.sql:293]: row_key & hash_diff are for incremental load

File: fum_flows/models/sources.yml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    database: |
+      {%- if  target.name == "dev" -%} anz-x-cosmos-prod-ccc3bd
+      {%- else -%} invalid_database
+      {%- endif -%}
+    schema: dragonfish_transaction_v1
+    tables:
+      - name: transaction_anz_classic
+      - name: transaction_feature_anz_classic
+      - name: transaction_anz_plus
+      - name: transaction_feature_anz_plus
```
Comment (nguyea57) [inline fum_flows/models/sources.yml:18]: transaction_feature_* are not needed

File: fum_flows/models/mart/fum_flow_whole_bank_current_vw.sql [hunk 3]
```diff
@@ -0,0 +1,3 @@
+{{ config(materialized='view') }}
+
+SELECT * FROM {{ ref("fum_flow_whole_bank") }}
```
Comment (nguyea57) [inline fum_flows/models/mart/fum_flow_whole_bank_current_vw.sql:3]: Need updating to the correct query

File: fum_flows/models/integration/fum_flow_whole_bank.sql (hunk 1 above)
Comment (nguyt120) [reply fum_flows/models/integration/fum_flow_whole_bank.sql:293]: vay xoa nha

File: fum_flows/models/integration/fum_flow_whole_bank.sql [hunk 4]
```diff
@@ -0,0 +1,290 @@
...
+        dst_ofi.fi_name dst_fi,
+        dst_product_code,
+        dst_sub_product_code,
+        dst_marketing_code,
+        dst_ofi.interest_rate dst_interest_rate,
+        dst_term,
+        transaction_amount,
+        all_retail_mfi_flag
+    FROM stg_transaction_whole_bank twb
+    JOIN stg_deposit_account_customer dac ON twb.src_account_number = dac.account_number
```
Comment (nguyea57) [inline fum_flows/models/integration/fum_flow_whole_bank.sql:190]: AND twb.src_product_code = dac.product_code AND twb.src_sub_product_code = dac.sub_product_code

File: fum_flows/models/integration/fum_flow_whole_bank.sql [hunk 5]
```diff
@@ -0,0 +1,290 @@
...
+    payid_transaction_count,
+    payid_transaction_amount,
+    interest_transaction_count,
+    interest_transaction_amount,
+    payto_transaction_count,
+    payto_transaction_amount,
+    deposit_withdrawal_transaction_count,
+    deposit_withdrawal_transaction_amount,
+    unidentified_transaction_count,
+    unidentified_transaction_amount,
```
Comment (nguyea57) [inline fum_flows/models/integration/fum_flow_whole_bank.sql:289]: remove last ","

File: fum_flows/models/integration/fum_flow_whole_bank.sql (hunk 4 above)
Comment (nguyt120) [reply fum_flows/models/integration/fum_flow_whole_bank.sql:190]: done

**Review by nguyea57 — APPROVED** [review]
LGTM

**Review by nguyt120 — COMMENTED** [review]
g

**Review by nguyed54 — APPROVED** [review]
LGTM Approved

**Pull request merged**